omod-1.10/src/test/java/org/openmrs/module/attachments/rest/AttachmentRestController1_10Test.java

```diff
@@ -121,8 +139,44 @@ public void saveAttachment_shouldUpdateObsComment() throws Exception {
 		assertEquals(editedComment, comment);
 	}
 	
+	@Test
+	public void shouldGetAttachment() throws Exception {
+		MockHttpServletRequest req = request(RequestMethod.GET, getURI() + "/" + getUuid());
+		SimpleObject result = deserialize(handle(req));
+		
+		Assert.assertEquals(getUuid(), PropertyUtils.getProperty(result, "uuid"));
+		Assert.assertNotNull(PropertyUtils.getProperty(result, "comment"));
+		Assert.assertNotNull(PropertyUtils.getProperty(result, "bytesMimeType"));
+		Assert.assertNotNull(PropertyUtils.getProperty(result, "bytesContentFamily"));
+	}
+	
+	@Test
+	public void shouldGetBytesMimeTypeOfAttachment() throws Exception {
+		MockHttpServletRequest req = request(RequestMethod.GET, getURI() + "/" + getUuid());
+		SimpleObject result = deserialize(handle(req));
+		
+		ComplexData complexData = obs.getComplexData();
+		
+		assertEquals(result.get("bytesMimeType"), complexDataHelper.getContentType(complexData));
+		assertEquals(result.get("bytesMimeType"), "application/octet-stream");
+	}
+	
+	@Test
+	public void shouldGetBytesContentFamilyOfAttachment() throws Exception {
+		MockHttpServletRequest req = request(RequestMethod.GET, getURI() + "/" + getUuid());
+		SimpleObject result = deserialize(handle(req));
+		
+		ComplexData complexData = obs.getComplexData();
+		
+		ContentFamily contentFamily = AttachmentsContext.getContentFamily(complexDataHelper.getContentType(complexData));
+		
+		assertEquals(result.get("bytesContentFamily"), contentFamily.toString());
+		assertEquals(result.get("bytesContentFamily"), "OTHER");
+	}
+	
```

Comment:
Look at existing methods in this class:
1. Use standard naming, such as `methodName_shouldDoThisAndThat`.
1. Use the given/when/then structure for each test case.